pkg/ccl/streamingccl/streamproducer/replication_manager_test.go

```diff
@@ -45,16 +46,20 @@ func TestReplicationManagerRequiresReplicationPrivilege(t *testing.T) {
 		execCfg = testTenants[0].ExecutorConfig().(sql.ExecutorConfig)
 	}
 
-	var sessionData sessiondatapb.SessionData
+	var m sessiondatapb.MigratableSession
 	var sessionSerialized []byte
 	tDB.QueryRow(t, "SELECT crdb_internal.serialize_session()").Scan(&sessionSerialized)
-	require.NoError(t, protoutil.Unmarshal(sessionSerialized, &sessionData))
+	require.NoError(t, protoutil.Unmarshal(sessionSerialized, &m))
```

Comment:
Will this work on a mixed-version cluster?

Reply:
yes, this will be fine, since adding new fields to a proto is backwards compatibile